backend/pkg/auth/auth_config.go

```diff
@@ -28,7 +28,13 @@ func (config AuthConfig) AuthString() string {
 }
 
 func (config AuthConfig) AuthURL() string {
-	return config.InjectAuthString()
+	authUrl := config.InjectAuthString()
```

Comment:
there's a bug, where if you append a slash to the URL the whole thing doesn't work.
This is slightly better?